backend/src/apiserver/client/argo_fake.go

```diff
@@ -28,7 +29,7 @@ func NewFakeExecClient() *FakeExecClient {
 }
 
 func (c *FakeExecClient) Execution(namespace string) util.ExecutionInterface {
-	if len(namespace) == 0 {
+	if len(namespace) == 0 && common.IsMultiUserMode() {
```

Comment:
Is this change really needed?
Understand in single-user mode, users don't need to pass namespace or experiment explicitly. But I think we found that we always find a  default experiment which belongs to namespace.

Reply:
You are right. This is no longer needed.